dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -406,12 +409,38 @@ impl CliDnsGroup {
 }
 
 #[derive(Debug, Args)]
-struct InstancesOptions {
+struct InstanceArgs {
+    #[command(subcommand)]
+    command: InstanceCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum InstanceCommands {
+    /// list instances
+    #[clap(alias = "ls")]
+    List(InstanceListArgs),
+    /// show detailed output for the selected instance.
+    ///
+    /// instances can be selected either by their UUID or by the instance and
+    /// project name.
```

Comment:
This comments suggests I could supply instance and project name, but I could not figure out how it parses that?